File: dsa/dsa/array/cpp/search_element.cpp
#include <iostream>
using namespace std;

int search(int arr[], int n, int x) {
    for(int i = 0; i < n; i++) {
        if (arr[i] == x)
            return i;
    }

    return -1;
}

int main() {
    int arr[] = {30, 50, 40, 5, 12, 222}, x= 12;
    cout<<"search index "<<search(arr, 4, x);
}

File: dsa_with_cp/dsa/array/python/search_element.py
def search(arr, element):
    for index in range(0, len(arr)):
        if arr[index] == element:
            return index

    return -1


if __name__ == "__main__":
    arr = [20, 5, 7, 25]
    element = 5

    print(search(arr, element))
